conf/broker.conf

```diff
@@ -159,6 +159,13 @@ skipBrokerShutdownOnOOM=false
 # Factory class-name to create topic with custom workflow
 topicFactoryClassName=
 
+# Max capacity of the topic name cache. 0 means unlimited cache.
```

Comment:
We'd better use -1 for unlimited cache and use 0 for no cache.

Reply:
Maybe the following design is better:
- `-1`: unlimited cache
- `0`: broker clears all cache per `maxSecondsToClearTopicNameCache`